### PR TITLE
docs: Synchronize generated sources with OAS markdown rendered as HTML for javadoc

### PR DESCRIPTION
**Pull request by JPPortier**

### CODE REVIEW

File: openapi-contracts/src/main/com/sinch/sdk/domains/verification/models/v1/webhooks/VerificationRequestEventResponseFlashCall.java [hunk 1]
```diff
@@ -28,7 +28,7 @@ public interface VerificationRequestEventResponseFlashCall
 
   /**
    * The phone number that will be displayed to the user when the flash call is received on the
-   * user&#39;s phone. By default, the Sinch dashboard will randomly select the CLI that will be
+   * user's phone. By default, the Sinch dashboard will randomly select the CLI that will be
```
Comment (asein-sinch) [inline openapi-contracts/src/main/com/sinch/sdk/domains/verification/models/v1/webhooks/VerificationRequestEventResponseFlashCall.java:31]: I've seen some other `&#39;` in the PR #90 which are not replaced in this PR:
 -  openapi-contracts/src/main/com/sinch/sdk/domains/verification/models/v1/VerificationMethod.java
 - openapi-contracts/src/main/com/sinch/sdk/domains/verification/models/v1/VerificationStatus.java

**Review by asein-sinch — DISMISSED** [review]

File: openapi-contracts/src/main/com/sinch/sdk/domains/verification/models/v1/webhooks/VerificationRequestEventResponseFlashCall.java (hunk 1 above)
Comment (JPPortier) [reply openapi-contracts/src/main/com/sinch/sdk/domains/verification/models/v1/webhooks/VerificationRequestEventResponseFlashCall.java:31]: Right.
This is coming from generator and dedicated redocly's enum descriptions injection
Working on it; thank you to have highlighted this

Comment (JPPortier) [reply openapi-contracts/src/main/com/sinch/sdk/domains/verification/models/v1/webhooks/VerificationRequestEventResponseFlashCall.java:31]: Finally: a false positive not important in this dedicated context ( `&#39;` or `'` US-ASCII does not matter here)
But you highlighted a missing point from generator: it was not handle correctly x-enumDescriptions coming from redocly markdown.
PR in review in dedicated repo to fix it; then source will be updated here

**Comment by JPPortier** [comment]
@asein-sinch according to https://github.com/sinch/sinch-sdk-java/pull/91#discussion_r1651034292
Last commit to sync with generated files done in this PR

**Review by asein-sinch — APPROVED** [review]

**Pull request merged**